torch/_inductor/decomposition.py

```diff
@@ -392,6 +386,12 @@ def bernoulli_(self, p=0.5):
     return self.copy_(torch.rand_like(self, dtype=torch.float32) < p)
 
 
+@register_extra_random_decomp([aten.bernoulli.p])
```

Comment:
?

Reply:
Hmmm let me try to find the original test failure that caused me to add this...

Context: I had a bernoulli test failing on inductor somewhere, and I noticed that inductor already does something similar for the inplace variant of bernoulli ([code](https://github.com/pytorch/pytorch/blob/fe0e28ab87ddda38a82efe74dd3eaf05e926f117/torch/_inductor/decomposition.py#L386)). I thought it would make sense to copy the pattern for the out-of-place variant, since it looks like the purpose of this decorator is precisely to handle the fact that randomness-related decomps can cause divergences from eager behavior.

Reply:
It was this test: `python test/inductor/test_torchinductor.py CpuTests.test_bernoulli1_cpu`. I'll confirm with @ngimel: does adding this decomp to inductor's "randomness" decomps sound like the right thing here?